### PR TITLE
Add external secrets in application chart

### PR DESCRIPTION
**Pull request by bilalbokharee**

### CODE REVIEW

**Comment by stakater-user** [comment]
@bilalbokharee Validation Successful

**Comment by rasheedamir** [comment]
@bilalbokharee data values-test.yaml so, pipeline can validate creation of this resource

File: application/values.yaml [hunk 1]
```diff
@@ -634,3 +634,44 @@ prometheusRule:
     #       for: 1m
     #       labels:
     #         severity: critical  
+##########################################################
+# External Secrets
+##########################################################
+externalSecret:
```
Comment (rasheedamir) [inline application/values.yaml:640]: so, we are taking input a list or a map? can a value in one environment be overridden by a value in another environment?

**Comment by stakater-user** [comment]
@bilalbokharee Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/02fa2349e063a724387787ece9a6bc617a65d69f/checks) has Failed!

**Comment by stakater-user** [comment]
@bilalbokharee Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/3d53dc18ef083ad9194663dc6ae63358f705465d/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful

File: application/values.yaml (hunk 1 above)
Comment (hanzala1234) [reply application/values.yaml:640]: we are taking as map now. it can be overridden

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**